.github/workflows/main.yml

```diff
@@ -3,16 +3,33 @@ name: Bedrock Node.js CI
 on: [push]
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - name: Run eslint
+      run: npm run lint
   test-node:
+    needs: [lint]
```

Comment:
Why not let tests run in parallel with lint?

Reply:
IMO it's a waste of time.  Linting errors should never be allowed.

Reply:
"waste of time"?  I'm confused since running in parallel *saves* time.  Linting errors result in a check error either way.  In the case where there are no linting errors, we want the tests to finish as fast as possible.

Reply:
The linting action takes 20s.  No need to run test suites that may take several minutes if linting fails.  Also, you're optimizing for a human clock.  In some cases, CI minutes are not free.

Reply:
I think we should optimize for human time.  I suspect lint CI failures are statistically uncommon compared to test failures.  Editor integrated linting and/or local lint runs help with that.  That's what I do.  Seems better to optimize for the common case of no lint issues where the tests would run either way, so the CI cost savings is negligible, and we might as well finish faster.  I've said enough. ;-)  I'm -1 on this serialization.  If we want to fight it out lets do so in another forum.